### PR TITLE
Bleach & Space Cleaner now cleans tiles & mobs that have been colored via the colorful reagent.

### PR DESCRIPTION
**Pull request by Irradiation**
Like it says on the tin.

[tweak][qol][gameplay]

:cl:
 * tweak: Space Cleaner can now clean up tiles and mobs affected by the colorful reagent.
 * tweak: Bleach can now clean up mobs affected by the colorful reagent.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -2092,6 +2092,9 @@
 		qdel(O)
 	else if(O.color && istype(O, /obj/item/weapon/paper))
 		O.color = null
+	if(O)
```
Comment (MadmanMartian) [inline code/modules/reagents/Chemistry-Reagents.dm]: Fix the logic on this, just remove the istype(paper) check above

Comment (Irradiation) [reply code/modules/reagents/Chemistry-Reagents.dm]: Fucking old coders

**Comment by Dacendeth** [comment]
Any chance to make paint remover do this too?

**Comment by Irradiation** [comment]
That's out of the purview of this PR, but sure. Though I can't remember if paint removers are rare or something.

**Comment by small-boss2** [comment]
paint removers are in the arts & craft crates in cargo

**Comment by FudgePucker** [comment]
Colorfulful reagent is really hard to make. It should be bleach only for both.

**Comment by Caboozles** [comment]
It’s a lot harder to clean given how much bleach you need to remove. If anyone can paint a little, they can paint a fuckload. You only need a tiny amount to begin shitting up the station so that it’s hard to see, and causing bartenders everywhere to kill themselves. 

Unless we make people who use color smoke valid on par with catbeasts. That’d be fine.

**Pull request merged**